lib/compact_index.rb

```diff
@@ -32,7 +32,7 @@ def self.version_line(version)
       deps = version[:dependencies].map do |d|
         [
            d[:gem],
-           d.version_and_platform.gsub(/, /, "&")
+           d.version_and_platform.split(', ').sort.join("&")
```

Comment:
Should I use double quotes or single quotes? There is both there and I'm not sure what is the expected way, since rubocop didn't complained :-)